### PR TITLE
Fix 32-bits warnings

### LINKED ISSUE #458: Build breaks on i386 after vtk-9.2.2 upgrade
```
/wrkdirs/usr/ports/graphics/f3d/work/f3d-1.3.1/library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:450:30: error: non-constant-expression cannot be narrowed from type 'unsigned int' to 'vtkIdType' (aka 'int') in initializer list [-Wc++11-narrowing]
        vtkIdType fId[2] = { face.mIndices[0], face.mIndices[1] };
                             ^~~~~~~~~~~~~~~~
/wrkdirs/usr/ports/graphics/f3d/work/f3d-1.3.1/library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:450:30: note: insert an explicit cast to silence this issue
        vtkIdType fId[2] = { face.mIndices[0], face.mIndices[1] };
                             ^~~~~~~~~~~~~~~~
                             static_cast<vtkIdType>( )
/wrkdirs/usr/ports/graphics/f3d/work/f3d-1.3.1/library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:450:48: error: non-constant-expression cannot be narrowed from type 'unsigned int' to 'vtkIdType' (aka 'int') in initializer list [-Wc++11-narrowing]
        vtkIdType fId[2] = { face.mIndices[0], face.mIndices[1] };
                                               ^~~~~~~~~~~~~~~~
/wrkdirs/usr/ports/graphics/f3d/work/f3d-1.3.1/library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:450:48: note: insert an explicit cast to silence this issue
        vtkIdType fId[2] = { face.mIndices[0], face.mIndices[1] };
                                               ^~~~~~~~~~~~~~~~
                                               static_cast<vtkIdType>( )
2 errors generated.
```

clang-14
FreeBSD 13.1

### PR DESCRIPTION
**Pull request by Meakk**
Fix #458

### CODE REVIEW

**Comment by mwestphal** [comment]
Nice fix but keep in mind that this can broke again at any point.

File: library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx [hunk 1]
```diff
@@ -442,20 +442,21 @@ class vtkF3DAssimpImporter::vtkInternals
 
       if (face.mNumIndices == 1)
       {
-        vtkIdType vId = face.mIndices[0];
+        vtkIdType vId = static_cast<vtkIdType>(face.mIndices[0]);
```
Comment (mwestphal) [inline library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:445]: Would there be a way to warn for cast like this, even on x64 ?

**Review by mwestphal — APPROVED** [review]

File: library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx (hunk 1 above)
Comment (Meakk) [reply library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:445]: No, we need to have a 32 bits CI, but I don't think it's high priority

Comment (mwestphal) [reply library/VTKExtensions/Readers/vtkF3DAssimpImporter.cxx:445]: yup

**Pull request merged**